### PR TITLE
Add JavaDoc to code

### PR DESCRIPTION
**Pull request by Lusi711**

### CODE REVIEW

File: src/main/java/zeronote/exceptions/InvalidIndexException.java [hunk 1]
```diff
@@ -1,11 +1,16 @@
 package zeronote.exceptions;
 
+//@@Lusi711
```
Comment (franceneee) [inline src/main/java/zeronote/exceptions/InvalidIndexException.java]: hehe lusi it's supposed to be ``@@author Lusi711`` xD I MADE A MISTAKE.

File: src/main/java/zeronote/exceptions/InvalidKeywordTagException.java [hunk 2]
```diff
@@ -1,5 +1,10 @@
 package zeronote.exceptions;
 
+//@@Lusi711
```
Comment (franceneee) [inline src/main/java/zeronote/exceptions/InvalidKeywordTagException.java]: ``@@author Lusi711`` same as the others xD

File: src/main/java/zeronote/exceptions/InvalidIndexException.java (hunk 1 above)
Comment (NeilBaner) [reply src/main/java/zeronote/exceptions/InvalidIndexException.java]: I fixed a few in my PR but others might still be wrong.

**Comment by NeilBaner** [comment]
@Lusi711 Once you resolve the merge conflicts, I think this PR is ready to be merged.

**Comment by franceneee** [comment]
heh, Lusi there are checkstyle issues xD

**Review by franceneee — APPROVED** [review]

**Pull request merged**